samples/tfm_hello_world/README.rst

```diff
@@ -0,0 +1,26 @@
+.. _tfm_hello_world:
+
+TF-M Hello World
+################
+
+Overview
+********
+
+A simple sample based on Hello World that demonstrates adding TF-M to an
+application.
+
+Building and Running
+********************
+
+This application works on the nRF9160 DK and the nRF5340 DK/PDK. Use the ns
```

Comment:
To be more future proof, state Trustzone enabled devices, and say "for instance 91 and 53" that way we don't need to update this again.

Reply:
It only works on the targets ported to TFM, so only the DKs, not e.g. thingy91.

I could write something like "on TFM-enabled boards like the nRF9160 DK or nRF5340 DK".

Reply:
Done.